gulpfile.js

```diff
@@ -137,4 +139,8 @@ gulp.task('patch', function() {
     runSequence('bump-patch', 'build', 'update-version');
 });
 
-gulp.task('build', ['css', 'js', 'css-min', 'js-min']);
+gulp.task('build', function() {
+    runSequence('lint', 'css', 'js', 'css-min', 'js-min', 'update-version')
```

Comment:
They both do the same thing. I know it's suboptimal and could be rewritten, but for now, they are independent - one processes files and puts them inside _build/_ whilst the second one processes files, minifies them and puts in _dist/_